src/main.js

```diff
@@ -116,14 +118,36 @@ var currentPoster;
 
 // event listeners go here 👇
 
+showMyPosterButton.addEventListener('click', function(e) {
+  e.preventDefault();
+  designPoster();
+  console.log("Hello!");
+});
+
 showRandomButton.addEventListener('click', updatePoster);
 
 window.addEventListener('load', updatePoster);
+
 makeYourOwnPosterButton.addEventListener("click", showFormSection);
 showSavedPostersButton.addEventListener("click", showSavedPostersSection);
 takeMeBackButton.addEventListener("click", leaveFormSection);
 backToMainButton.addEventListener("click", leaveSavedPostersSection);
 
+// functions and event handlers go here 👇
+
+
+function designPoster() {
+  new Poster(image, title, quote);
```

Comment:
I am wondering if we want to put this new instance in the current poster variable that is set up for us on line 117 or if there is no use for that yet.